nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/custom/BaseAdjustContrast.java

```diff
@@ -0,0 +1,19 @@
+package org.nd4j.linalg.api.ops.custom;
+
+import org.nd4j.base.Preconditions;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.api.ops.DynamicCustomOp;
+
+public abstract class BaseAdjustContrast extends DynamicCustomOp {
+    public BaseAdjustContrast() {
+    }
+
+    public BaseAdjustContrast(INDArray in, double factor, INDArray out) {
```

Comment:
you also need SDVariable-based constructors here, as other ops have

Reply:
Added.

Reply:
Done.